### PR TITLE
remove deprecated changesex

### PR DESCRIPTION
**Pull request by mekolat**
Instead of using a builtin, I made it work the same way as for classes

## goes with https://github.com/themanaworld/tmwa/pull/58

### CODE REVIEW

File: world/map/npc/002-2/eurni.txt [hunk 1]
```diff
@@ -27,7 +24,7 @@ L_Change:
     if (Zeny < 10000)
         goto L_NoMoney;
     set Zeny, Zeny - 10000;
```
Comment (mekolat) [inline world/map/npc/002-2/eurni.txt]: @wushin the npc do not tell the player that it will take 10k zeny.. should this be changed?

**Comment by mekolat** [comment]
@wushin I suggest we lower this: https://github.com/themanaworld/tmwa-server-data/pull/331/files#diff-7a423ee3a0d529f1c041aaec0faccbacL5

to something between 100~500

**Comment by nashley** [comment]
@mekolat Lowering it to 1k sounds reasonable to me.

**Comment by mekolat** [comment]
But why should it even cost anything to change the sex if it's free to change hair?

**Pull request merged**